samples/notebooks/python/1-basic-loading-the-kernel.ipynb

```diff
@@ -0,0 +1,235 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
```

Comment:
this doesn't look right. same in other notebooks

Reply:
fixed!